msteams-platform/concepts/authentication/nested-authentication.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Nested app authentication
+description: Learn about nested authentication in the Microsoft Teams platform.
+ms.date:
+ms.topic: conceptual
+author: v-ypalikila
+ms.author: surbhigupta
+ms.localizationpriority: medium
+---
+
+# Nested app authentication
+
+> [!NOTE]
+> Nested app authentication is available only in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+
+Nested app authentication (NAA) is a new authentication protocol for single page applications that are embedded in host environments like Teams, Outlook, and Microsoft 365. It simplifies the authentication process and
```

Comment:
"Single page application" is a general term that implies "Tab" in the MetaOS/Teams app world, but I think we should be more explicit NAA is for "Tabs" only. It does not work for other app "elements/capabilities" like bots or Message Extensions.

Reply:
Added a sentence in the Note that "Nested app authentication is only supported in tabs."